cms/urls.py

```diff
@@ -17,6 +17,11 @@
 admin.site.site_header = _('Studio Administration')
 admin.site.site_title = admin.site.site_header
 
+if settings.PASSWORD_POLICY_COMPLIANCE_ROLLOUT_CONFIG.get('ENABLE_COMPLIANCE_CHECKING', False):
+    from openedx.core.djangoapps.password_policy.forms import PasswordPolicyAwareAdminAuthForm
+    admin.site.login_form = PasswordPolicyAwareAdminAuthForm
```

Comment:
It may be better to define a custom edx-specific AdminSite somewhere else so that urls.py doesn't need to care about this.

Reply:
That long if condition... Maybe make it a utility call like like complience_checking_enabled() or something so that strings and defaults don't have to be specified every check. But that's just a nit.